cli/tests/external-db-tests/timescale_test.go

```diff
@@ -0,0 +1,33 @@
+package external_db_tests
+
+import (
+	"fmt"
+	test_utils "github.com/timescale/tobs/cli/tests/test-utils"
+	"testing"
+)
+
+func TestTimescale(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping TimescaleDB tests")
+	}
+
+	fmt.Println("Performing TimescaleDB tests for external db setup...")
+	releaseInfo := test_utils.ReleaseInfo{
+		Release:   RELEASE_NAME,
+		Namespace: NAMESPACE,
+	}
+
+	//releaseInfo.TestTimescaleGetPassword(t, "")
```

Comment:
are these important?

Reply:
These are the tests to check change password for TimescaleDB, As mentioned earlier in the test suite as TimescaleDB is in the same cluster. The below call is failing

```
pool, err = pgxpool.Connect(context.Background(), dbURI)
if err != nil {
  return nil, err
}
```

Error:

```
Error: could not change TimescaleDB password: failed to connect to `host=external-db-tests.default.svc.cluster.local user=postgres database=postgres`: hostname resolving error (lookup external-db-tests.default.svc.cluster.local: no such host)
```

This works with forge. but some issue with the local TImescale DNS resolution.